### PR TITLE
Add installation instructions

### PR DESCRIPTION
**Pull request by davidgasquez**
Hey there! Saw the installation were missing on the README.

### CODE REVIEW

**Comment by davidgasquez** [comment]
@d70-t, any thoughts on this?

File: README.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
 
 A readonly implementation of fsspec for IPFS.
 
+## Installation
+
+You can install `ipfsspec` directly from git with the following command:
+```bash
+pip install git+https://github.com/fsspec/ipfsspec
```
Comment (d70-t) [inline README.md:9]:
```suggestion
pip install ipfsspec
```

**Review by d70-t — COMMENTED** [review]
Hey @davidgasquez, thanks for the initiative! I've been away for the last 5 weeks and just slowly catching up, thus answering only that late.

I guess the official way of installing would be to install the latest released version via pypi, thus just using the package name in the command. Your suggestion would install the current (maybe unreleased) `main` branch.

**Comment by d70-t** [comment]
I've had a look into the CI errors above. The are seemingly related to updates to common IPFS servers which had not yet been accounted for in the code. I've added a few updates to the current `main`. If you'd merge or rebase, the errors should go away.

**Comment by davidgasquez** [comment]
Thanks for jumping in!

> I guess the official way of installing would be to install the latest released version via pypi, thus just using the package name in the command. Your suggestion would install the current (maybe unreleased) main branch.

Agree! 

> I've added a few updates to the current main. If you'd merge or rebase, the errors should go away.

:heavy_check_mark:

**Comment by d70-t** [comment]
Thanks for updating! 🎉

**Pull request merged**